academy/3-my-own-chain/node-api-and-cli.md

```diff
@@ -243,25 +243,57 @@ It helps to understand the concepts clearly when working hands-on with the Cosmo
 
 You can also inspect your keys. These are held in the backend keyring, which by default is that of the operating system:
 
+<CodeGroup>
+
+<CodeGroupItem title="Local">
+
 ```sh
 $ ./simd keys list
 ```
 
-As you might have expected, you do not have any keys yet:
+</CodeGroupItem>
 
+<CodeGroupItem title="WSL2">
+
+```sh
+$ ./simd keys list --keyring-backend test
```

Comment:
@julienrbrt It is possible we need to do this to cater to those who do it on WSL2.

Reply:
What's WSL2?

Reply:
the Windows Subsystem for Linux
https://learn.microsoft.com/en-us/windows/wsl/about